apps/nuxt3-ssr/components/content/TreeNode.vue

```diff
@@ -1,5 +1,5 @@
 <script setup lang="ts">
-import type { IOntologyItem } from "metadata-utils";
+import type { IOntologyItem } from "~/interfaces/types";
```

Comment:
Since ontologies are also used elsewhere, shouldn't this one stay? Or are the catalogue ontologies special?